NitroxServer/ConsoleCommands/ListCommand.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.GameLogic;
+using NitroxModel.Logger;
+
+namespace NitroxServer.ConsoleCommands
+{
+    class ListCommand : Command
+    {
+        private PlayerManager playerManager;
+
+        public ListCommand(PlayerManager playerManager)
+        {
+            Name = "list";
```

Comment:
This might be cleaner using the base constructor:

```
public ListCommand() : base("list")
```

or you can simply use reflection to get the name of the current class.